docs/buildah-from.md

```diff
@@ -15,6 +15,10 @@ The container ID of the container that was created.  On error, -1 is returned an
 
 ## OPTIONS
 
+**--creds** *creds*
```

Comment:
Right.  I think that to make life simpler, I'm just going to get credentials from the --creds param for all the functions that either need source or destination credentials and then let the functions do the right thing for the user.